.github/workflows/release.yml

```diff
@@ -1,14 +1,17 @@
 # This sets the environment for building Kiwix JS store packages and runs the build script
 
-name: Build all packages on release publish
+name: Build all packages nightly or on publish new release
```

Comment:
As it covers also the nightly builds, maybe the YAML file should not be named "release.yml"

Reply:
OK, I could rename it `nightly_or_release.yml`,

Reply:
OK